openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,35 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A collection of one or more assertions about a subject, provided by a Credential Issuer. These assertions are also known as claims.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Verifiable Credential (or Credential):
+:  An Issuer-signed Credential Dataset whose integrity can be cryptographically verified. It can be of any Credential Format.
 
-W3C Verifiable Credential:
-:  A Verifiable Credential compliant to the [@VC_DATA] specification.
+Credential Format:
+:  A format used to represent a Verifiable Credential. The exact parameters required to describe a Credential Format are defined in the Credential Format Profile.
+
+Credential Format Profile:
+:  Sets of parameters specific to individual Credential Formats. Credential Format Profiles for W3C Verifiable Credentials [@VC_DATA] and ISO/IEC 18013-5 mDL [@ISO.18013-5] are provided in (#format_profiles). Other specifications or deployments can define their own Credential Format Profiles using the extension points defined in this specification.
+
+Credential Configuration:
+:  A Credential Configuration consists of a Credential Format including a type, information about requesting Credentials using this Credential Configuration (in particular, a `scope` value), information on cryptographic methods and algorithms supported for issuance, and display information to be used by the Wallet. A Credential Configuration is identified by a unique string.
```

Comment:
```suggestion
:  A Credential Configuration describes a particular type of Credential that a Credential Issuer is offering to issue. It contains a Credential Format and specifies the corresponding parameters given in the Credential Format Profile. Furthermore it includes information about requesting Credentials using this Credential Configuration, information on cryptographic methods and algorithms supported for issuance, and display information to be used by the Wallet. A Credential Configuration is identified by a Credential Configuration Identifier string that is unique to an Issuer.
```

Reply:
I think your first sentence does not capture the essence of a cred. configuration (as it is used right now).

Reply:
I felt it was missing a sentence on the meaning/motivation instead of just listing the contents. whats missing from your view? type is overloaded, I know..